controllers/grid/navigationMenus/form/NavigationMenuForm.inc.php

```diff
@@ -88,6 +88,13 @@ function fetch($request) {
 			return $a->getId() - $b->getId();
 		});
 
+		foreach ($unassignedItems as $unassignedItem) {
+			import('classes.core.ServicesContainer');
+			$navigationMenuItemTypes = ServicesContainer::instance()
```

Comment:
I don't think `transformNavMenuItemTitle` has a return value, so the assignment here is probably unnecessary?